Anchor.toml

```diff
@@ -1,6 +1,6 @@
 [toolchain]
 anchor_version = "0.29.0"
-solana_version = "1.17.13"
+solana_version = ">=1.17.13,<=2.0.0"
```

Comment:
Do you know if Anchor supports this syntax? I don't think it does, so unless you're sure that it works, we should probably keep this as is